bundles/org.openhab.binding.homewizard/src/main/java/org/openhab/binding/homewizard/internal/devices/kwh_meter/HomeWizardKwhMeterHandler.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.homewizard.internal.devices.kwh_meter;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.homewizard.internal.HomeWizardBindingConstants;
+import org.openhab.binding.homewizard.internal.devices.HomeWizardEnergyMeterHandler;
+import org.openhab.binding.homewizard.internal.devices.energy_socket.HomeWizardEnergySocketMeasurementPayload;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.Thing;
+
+/**
+ * The {@link HomeWizardKwhMeterHandler} implements functionality to handle a HomeWizard kWh Meter.
+ *
+ * @author Gearrel Welvaart - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class HomeWizardKwhMeterHandler extends HomeWizardEnergyMeterHandler {
+
+    /**
+     * Constructor
+     *
+     * @param thing The thing to handle
+     *
+     */
+    public HomeWizardKwhMeterHandler(Thing thing) {
+        super(thing);
+        supportedTypes.add("HWE-KWH1");
+        supportedTypes.add("SDM230-wifi");
+        supportedTypes.add("HWE-KWH3");
+        supportedTypes.add("SDM630-wifi");
```

Comment:
see earlier comment about naming convention (just another reminder to also adapt it here)